server/src/modules/queues/queue.js

```diff
@@ -0,0 +1,23 @@
+const { Queue } = require("bullmq");
+const { QUEUE_EVENTS } = require("./constants");
+const redisConnection = { host: "localhost", port: 6379 };
```

Comment:
We should get those values from environment variables, for the `host` and `port`.